### PR TITLE
feature: periodically store operational info

### PR DESCRIPTION
**Pull request by alessandroberlati**
## Description

This PR changes the way operational info are processed. Instead of relying on a celery task called directly after the authorization, we send the serialized data in a redis queue. The operational info are then periodically retrieved and stored in mongo.

## Checklist

- [x] I have followed the indications in the [CONTRIBUTING](https://github.com/immuni-app/immuni-backend-analytics/blob/master/CONTRIBUTING.md).
- [x] The documentation related to the proposed change has been updated accordingly (plus comments in code).
- [x] I have written new tests for my core changes, as applicable.
- [x] I have successfully run tests with my changes locally.
- [x] It is ready for review! :rocket:

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -47,20 +47,11 @@ services:
     build:
       <<: *common-build
       target: worker
-  worker_exposure_payload:
+  worker_scheduled:
```
Comment (ocardia) [inline docker/docker-compose.yml:50]: Is it all scheduled?

File: immuni_analytics/celery/scheduled/tasks/store_operational_info.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@celery_app.task()
+def store_operational_info() -> None:  # pragma: no cover
+    """
+    Celery doesn't support async functions, so we wrap it around asyncio.run.
+    """
+    asyncio.run(_store_operational_info())
```
Comment (ocardia) [inline immuni_analytics/celery/scheduled/tasks/store_operational_info.py:30]: Maybe worth creating an `async_celery_task` decorator which takes the `celery_app` as an argument?

Then we'd have something like:
```python
@async_celery_task(celery_app)
async def _store_operational_info() -> None:
    """
    Retrieve up to a fixed number of operational info and save it into mongo.
    """

    _LOGGER.info("Store operational info periodic task started.")
```

File: immuni_analytics/core/config.py [hunk 3]
```diff
@@ -79,6 +82,11 @@
     cast=validate_crontab("STORE_INGESTED_DATA_PERIODICITY"),
     default="* * * * *",
 )
+STORE_OPERATIONAL_INFO_PERIODICITY: str = config(
+    "STORE_OPERATIONAL_INFO_PERIODICITY",
+    cast=validate_crontab("STORE_OPERATIONAL_INFO_PERIODICITY"),
+    default="* * * * *",
```
Comment (ocardia) [inline immuni_analytics/core/config.py:92]: This is every minute right?

Comment (alessandroberlati) [reply immuni_analytics/core/config.py:92]: correct

File: docker/docker-compose.yml (hunk 1 above)
Comment (alessandroberlati) [reply docker/docker-compose.yml:50]: What do you mean? 🤔

File: immuni_analytics/celery/scheduled/tasks/store_operational_info.py (hunk 2 above)
Comment (alessandroberlati) [reply immuni_analytics/celery/scheduled/tasks/store_operational_info.py:30]: I would tackle this in another pr.

File: docker/docker-compose.yml (hunk 1 above)
Comment (ocardia) [reply docker/docker-compose.yml:50]: Whether all the tasks that run within this worker are scheduled? (I guess so, just double-checking.)

Comment (alessandroberlati) [reply docker/docker-compose.yml:50]: Ah, yes they are : D

**Review by ocardia — APPROVED** [review]

**Pull request merged**